### DISCUSSION TITLE
关于Waline的UI的一个对照

### DISCUSSION THREAD

**Discussion by Pathsis** [walinejs/waline#669] [Ideas]
总觉得Waline现在的UI不是很好，我发现一个评论系统的UI还是很不错的，它已经停止了开发，我们是否可以借鉴下它的布局和风格呢？

https://just-comments.com/

<img width="904" alt="截屏2021-11-18 01 35 22" src="https://user-images.githubusercontent.com/72077265/142252661-33c1c018-ea99-4254-92f0-b96ba8727bed.png">

**Comment by Mister-Hope** [comment]
Waline更倾向于一个页尾挂件，所以从基本的出发点来说，官方客户端面向一个博客文章或文档文尾评论的场景，不太适合这种占用大量空间的评论框布局格式。这也是为什么我们选择窄的页眉和页脚来放置功能按钮及信息输入区。采用宽松的布局以及相关指导语的确会有助于用户体验，但同时也增加了它的侵入性（一方面破坏页面布局，另一方面可能会跟页面其他元素风格产生冲突）

**Reply by Pathsis** [reply]
感谢回复。抱歉回复晚了！我不是很理解“有助于用户体验”和“增加入侵性”如何权衡。我更倾向于“有助于用户体验”。我现在的感觉是，Waline的UI放在多数博客中都特别“突出”（丑是真的丑），视觉上的“入侵”性不知道算不算入侵呢。

至于指导语，您坚持“欢迎评论”比“您无需登录或填写任何个人信息而可直接匿名评论”更合适，我必须得尊重您的意见，同时仍然向您推荐。

**Reply by AnzhiZhang** [reply]
你可以通过locale自定义placeholder
